doc/datamodel/compartments.md

```diff
@@ -0,0 +1,53 @@
+Compartment report data model {#data_compartments}
+============
+
+## Prerequisites
+
+A _neuron_ is a single nerve cell simulated, and connected with thousands of
+other neurons. It is constructed from a morphological skeleton aka _morphology_.
+This skeleton has a tree-like structure originating at the soma, the cell body
+of each neuron. Each neuron has a unique _GID_ (global identifier). Every neuron
+represents an instance of a certain morphology in 3D space which is link in the
+MVD circuit description.
+
+The branches of the tree are formed by _sections_ which are subdivided into
+smaller pieces. They start at the soma and eventually branch into multiple
+sections in a tree-like structure.
+
+During the reconstruction from real neurons, each section is formed by a number
+of _segments_. Each segment has two endpoints in 3D space, as well as a radius
+for each endpoint. These segments consequently represent tubelets in 3D space.
+
+For the simulation, each section is decomposed into m _compartments_, which have
+a n : m mapping to segments. For each compartment, various observables are
+simulated and may be reported in a CompartmentReport.
+
+## Meta data
+
+* list of neuron GIDs
+* start time in ms
+* end time in ms
+* dt in ms
+* data unit as string, e.g. mV, pA
+
+## Compartment information
+
+* for each neuron, sorted ascending by GID
+* number of compartments of each section per neuron, sections order defined by
+  morphology
+* all sections of a neuron are always described, i.e. unreported sections have
+  zero compartments
+
+Compartments are always linearly interpolated along all segments of a section,
+i.e. compartments have all the same length within a particular section.
```

Comment:
I really don't understand it, to me the sentences are not equivalent.

What does linear interpolation mean for you here? To me interpolating is finding a function which is constrained to pass through a set of given sample points (which typically come from a continuous function we want to approximate but is not fully known). What's the function being interpolated here, what are the control points?

Could Jim mean that since the variables measured at compartments are continuous, compartments could be understood as sample points that are equally spaced along a section (i.e., the sample points are placed at the center of the compartments), and that you could do linear inteporlation along a section to find the approximate value of the variable at the segments?